nix/build-all.py

```diff
@@ -325,44 +343,51 @@ def run_cmake(arg1, cmake_args, cmake_dir=None, cwd=None):
 
 def git_clone_or_pull_repository(clone_url, target_dir, revision=None):
     """Lazily clones the `git` repository denoted by `clone_url` into
-    the `target_dir` or pulls latest changes if the `target_dir` exists (naively assumes
-    that a working clone exists there) and optionally checks out a revision
-    `revision` after cloning or in the existing clone if `revision` is not
-    `None`."""
+        the `target_dir` or pulls latest changes if the `target_dir` exists (naively assumes
+        that a working clone exists there) and optionally checks out a revision
+        `revision` after cloning or in the existing clone if `revision` is not
+        `None`."""
     if not os.path.exists(target_dir):
         logger.info("cloning '%s' into '%s'" % (clone_url, target_dir))
         run([git, "clone", clone_url, target_dir])
     else:
         logger.info("directory '%s' already cloned. Pulling latest changes." % (target_dir,))
-        run([git, "pull", clone_url], cwd=target_dir)       
-        
-    if revision != None:
-        run([git, "checkout", revision], cwd=target_dir)
+        run([git, "pull", clone_url], cwd=target_dir)
+
+if revision != None:
+    run([git, "checkout", revision], cwd=target_dir)
 
-def build_dependency(name, mode, build_tool_args, download_url, download_name, download_tool=download_tool_default, revision=None, patch=None, additional_files={}, no_append_name=False):
+def python_system_homedir(name, mode, build_tool_args, download_url, download_name, download_tool=download_tool_default, revision=None, patch=None, additional_files={}, no_append_name=False):
```

Comment:
Was renaming this function intentional? I find the new name less clear.

Reply:
Ohhh, this is stupid replace-all error. I don't know how this has happened....
This should be changed back to build_dependency. I can commit new verson, ok?

Reply:
This has been changed back to build_dependency. I will commit new version.